hlx_statics/blocks/embed/embed.css

```
@@ -2,6 +2,10 @@ main div.embed-container:has(.embed.background-color-white) {
   background-color: white;
 }
 
+main div.embed-wrapper {
+  padding: 20px 0;
```

```suggestion
  padding: 20px 0;
```
 
Please remove for now. We will fix this (likely in a similar way) as part of [DEVSITE-1590](https://jira.corp.adobe.com/browse/DEVSITE-1590), see #2 in description